test/e2e/workflow/deployment_test.go

```diff
@@ -361,7 +361,7 @@ var _ = Describe("NetworkAddonsConfig", func() {
 			})
 		})
 
-		Context("with an overridden configuration", func() {
+		Context("with a non-default configuration", func() {
```

Comment:
I think this belongs to the former commit, can you move it please?

Reply:
Done.